new_relic_scripts/totp_sign_in.js.erb

```diff
@@ -0,0 +1,151 @@
+var crypto = require('crypto');
+var expect = require('chai').expect
+var url = require('url')
+
+var IDP_URL = '<%= ENV['IDP_URL'] %>'
+var EMAIL =  '<%= ENV['NR_TOTP_SIGN_IN_EMAIL'] %>'
+var PASSWORD = '<%= ENV['NR_TOTP_SIGN_IN_PASSWORD'] %>'
+var TOTP_SECRET = '<%= ENV['NR_TOTP_SIGN_IN_TOTP_SECRET'] %>'
+
+// Initialize $browser and $driver if we are local and not on New Relic
+try {
+  $browser && $driver
+} catch (error) {
+  global['$driver'] = require('selenium-webdriver');
+  global['$browser'] = new $driver.Builder().forBrowser('chrome').build();
+}
+
+// Ref: https://github.com/guyht/notp/blob/master/index.js
+function generateTOTP() {
```

Comment:
since we're already pre-processing this file, would it make sense to inline the source of the `notp` gem? ex something like:

```erb
(function(module, exports, require) {
  <%= File.read('node_modules/notp/index.json') %>
})(module, exports, require)
```

Reply:
Thought about that. I decided to do it this way for now with plans to use something like webpack to bundle it later